tests/testsuite/directory.rs

```diff
@@ -683,3 +683,44 @@ fn workspace_different_locations() {
 ",
         ).run();
 }
+
+#[test]
+fn version_missing() {
```

Comment:
In general the error messages are not nearly adequately tested. I'd like one day to be sure we have all the code branches tested. Thank you for adding this test!